### ISSUE TITLE
Need Last.fm Scrobbler and Sound Cloud Scrobbler to both work on Chrome

### ISSUE THREAD

**Issue by knik23** [web-scrobbler/web-scrobbler#1527]
### Steps to reproduce (for bugs)

1.
2.
3.

### Steps for resolving the issue

- [ ] Reauthenticate on Last.fm

- [ ] Try to use unpacked extension from latest master

- [ ] Check other issues for duplicates

### Environment

* Version used:
* Browser name and version:

**Comment by knik23** [comment]
Noticed you have fixed it for SoundClound.

**Comment by inverse** [comment]
Not sure what the issue is your describing? 

Could you clarify please.

**Comment by knik23** [comment]
Can't scrobbler SoundCloud in same scrobbler as last.fm on chrome. You have to do separately

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by knik23** [comment]
Want to mashup them like music video editing for movie. ;)

That's what I've been doing for last three months straight. Almost finished with tonight going to premier at Cannes.



Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by alexesprit** [comment]
I still can't understand the issue, but I'll try to help you.

I suppose SoundCloud Scrobbler is [that](https://chrome.google.com/webstore/detail/soundcloud-scrobbler/egbaojncijacajndhncghmjeielgocjd) extension, and you want to scrobble music from SoundCloud via that extension and scrobble music from other websites via Last.fm Scrobbler.

Last.fm Scrobbler provides a feature to disable supported websites. You should open the options page and turn off SoundCloud support.

If you experience an issue related to SoundCloud Scrobbler extension, you should contact to authors of this extension. We don't support that one.

**Comment by knik23** [comment]
I figured out what was going on.  Sometimes I was using Windows browsers which changes your profile a bit the bands listed for example. Finally I realized wasn’t using Chrome, erm.. Haven’t been on since I figured that’s what my issue. Now just need type extensions for other Music websites used.

If I have any questions I’ll send message if that’s alright. Don’t want to be a bother.  I know you are connected with David that’s the one I’m using – he probably sent you to me because he didn’t want to deal with me anymore. But I did donate 25$. Thank you both – Just too much was going on at the time when I was first trying GitHub.

Have a Happy New Year…

Kindly,
-Angela

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10
